deploy/gallery/instructions.md

```diff
@@ -1,23 +1,21 @@
-If you've landed on this page from Synapse gallery then Congrats! -- you are in the right place.
+If you've landed on this page from Synapse gallery, we assume you have the infrastructure ready and have generated the custom vision model v2 package. If this is true proceed else follow the [readme](https://github.com/Azure/Azure-Orbital-Analytics-Samples/blob/main/deploy/README.md) file to deploy the infrastructure.
```

Comment:
We assume that you have the infrastructure ready for the pipeline execution and have generated the custom vision model v2 package.

If this is true then proceed to next section, otherwise proceed as suggested in the following sections in [readme](https://github.com/Azure/Azure-Orbital-Analytics-Samples/blob/main/deploy/README.md) to setup the infrastructure and package the pipeline.
- [Prepare to run infrastructure deployment](https://github.com/Azure/Azure-Orbital-Analytics-Samples/blob/main/deploy/README.md#preparing-to-execute-the-script)
- [Deploy Infrastructure](https://github.com/Azure/Azure-Orbital-Analytics-Samples/blob/main/deploy/README.md#deployment-of-infrastructure-using-bicep-template)
- [Configure the infrastructure resources](https://github.com/Azure/Azure-Orbital-Analytics-Samples/blob/main/deploy/README.md#configuring-the-resources)
- [Package the pipeline](https://github.com/Azure/Azure-Orbital-Analytics-Samples/blob/main/deploy/README.md#packaging-the-synapse-pipeline)

Reply:
I have updated the comments but will wait for #86 to merge first.